05-UI-ReactJS/source/movie.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import {Link} from "react-router";
+
+class Movie extends React.Component{
+  constructor () {
+    super();
+    this.state = {title:"",
+    year:"",
+    duration:"",
+    favourite: false,
+    }
+  }
+
+  render () {
+    let item = {title: "", year: "", duration: "", favourite: false};
+    return (
+      <div className="movie">
+        <Link to={`MovieInput/0`}><button className="buttonSelect">Movie Input</button></Link>
```

Comment:
This is semantically wrong. You should wrap a button withe the link, you just need the Link and could set the class to the  Link.